lib/ansible/modules/windows/win_shortcut.ps1

```diff
@@ -18,6 +19,7 @@ $spec = @{
         icon = @{ type='path' }
         description = @{ type='str' }
         windowstyle = @{ type='str'; choices=@( 'maximized', 'minimized', 'normal' ) }
+        run_as_admin = @{ type='bool'; default=$false }
```

Comment:
I was confused by this parameter name, thinking 'why not use become instead'.
But that's not the purpose. Wonder if we can use a better name for this.

Also, which admin will it run as ? We have the typical "Administrator" user disabled.

Reply:
I was struggling to come up with a good name for this, the flag itself it relates to is the [RunAsUser](https://msdn.microsoft.com/en-us/library/dd891314.aspx) flag but even that isn't an accurate description of what it does. It really relates to the `Run as administrator` checkbox you can find in the Advanced properties

![image](https://user-images.githubusercontent.com/8462645/48450159-f85fdf00-e7f0-11e8-89f2-70ce6cc01c3c.png)

Basically if the shortcut targets an executable and this flag is set, it will open the shortcut like someone had right clicked -> `Run as administrator`. It's merely a convenience factor to ensure that whoever opens the shortcut will result in the executable being run with administrator privileges.

Reply:
Ok, leave it like that !